pandas/tests/frame/test_convert_to.py

```diff
@@ -281,17 +281,20 @@ def test_to_records_datetimeindex_with_tz(self, tz):
         # both converted to UTC, so they are equal
         tm.assert_numpy_array_equal(result, expected)
 
-    def test_to_dict_box_scalars(self):
-        # 14216
+    @pytest.mark.parametrize('orient,item_getter',
+                             [('dict', lambda d, col, idx: d[col][idx]),
+                              ('records', lambda d, col, idx: d[idx][col]),
+                              ('list', lambda d, col, idx: d[col][idx]),
+                              ('split', lambda d, col, idx: d['data'][idx][d['columns'].index(col)]),
+                              ('index', lambda d, col, idx: d[idx][col])
+    ])
```

Comment:
Ah, that's one way to parameterize 🙂 

Could you reformat so that it looks something like this:

~~~python
@pytest.mark.parametrize('orient,item_getter', [
    ('dict', lambda d, col, idx: d[col][idx]),
    # etc.
~~~

Also, comment above the decorator here explaining what these parameters mean.